package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "since",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
Typically we bump this directly on `dev` when we're about to cut over to `master`.

Reply:
Roger that. Wasn't sure of the workflow but it should be fine to have the version bump, even if a little premature now. Noted for next time.